templates/blog/topics/maas.html

```diff
@@ -0,0 +1,46 @@
+{% extends "templates/one-column.html" %} 
+
+{% block title %}MAAS{% endblock %}
+
+{% block meta_description %}All the latest news on MAAS (Metal as a Service) - the smartest way to manage bare metal.{% endblock %}
+
+{% block content %} 
+
+  <section class="p-strip--accent p-blog-topic__strip is-dark is-shallow u-image-position" style="overflow: hidden; background-color: #333">
+    <div class="row header-strip u-equal-height">
+      <div class="col-6">
+        <div class="p-heading-icon">
+          <div class="p-heading-icon__header">
+            <img class="p-blog-heading__icon" src="{{ASSET_SERVER_URL}}0228cce3-product-page-maas-logo.svg" alt=""/>
+          </div>
+          <p class="p-heading--three">Metal as a Service</p>
+          <p>All the latest news on MAAS – the smartest way to manage bare metal.</p>
+        </div>
+        <p>
+          <a class="p-link--external p-link--inverted" href="https://maas.io">Learn more about MAAS</a>
+        </p>
+      </div>
+      <div class="col-5 prefix-1">
+        <img
+          src="{{ASSET_SERVER_URL}}e6c6b98f-Maas-Visual-Screen.png?w=576"
+          class="u-hidden--small u-image-position--bottom"
+          style="padding-top: 1rem; max-height: 100%;"
+          alt="" />
+      </div>
+    </div>
+  </section>
+
+  <section class="p-strip is-shallow{% if current_page and current_page == 1 and upcoming_events %} is-bordered{% endif %}" id="posts-list">
```

Comment:
I don't think we need `{% if current_page and current_page == 1 and upcoming_events %} is-bordered{% endif %}`, we're not displaying `upcoming_events`, and I don't think they're included anywhere besides the blog index, so this will never evaluate as true. Also, I can't see an instance on https://blog.ubuntu.com/topics/maas where the border is applied.